src/main/java/tech/gruppone/stalker/server/model/db/OrganizationDao.java

```diff
@@ -27,9 +27,8 @@
   @Column("description")
   String description;
 
-  @Builder.Default
   @Column("isPrivate")
-  Boolean isPrivate = false;
+  Long isPrivate;
```

Comment:
come mai l'hai cambiato? non funzionava? comunque ricordiamoci di cambiarlo adeguatamente dopo il refactor dell'api

Reply:
Sempre per il discorso che in create.sql ora isPrivate è un tinyint(1), quindi o è 0 o e 1 (non è un booleano)